packages/zoltan/src/CMakeLists.txt

```diff
@@ -400,6 +400,7 @@ APPEND_SET(SOURCES
   rcb/inertial3d.c
   rcb/point_assign.c
   rcb/rcb_box.c
+  rcb/rcb_partition_tree.c
```

Comment:
I changed this to zoltan_partition_tree.h and currently have it in the include folder but not sure if that will be the desired place for this? If this will share both rcb and phg code maybe a new folder called partitionTree could contain this.